tests/unittest/llmapi/apps/test_responses_streaming_tool_calls.py

```
@@ -0,0 +1,198 @@
+        ):
+            result = _should_send_done_events(
+                output=output,
+                output_index=0,
+                tool_parser_id="test_parser",
+                tools=[],
+                tool_parser_dict={0: None},
+                streaming_events_helper=helper,
+                finished_generation=False,
+            )
+
+        should_reasoning, should_text, reasoning_content, text_content, done_tool_calls = result
```

Ruff (0.16.1)

[warning] Unpacked variable `should_reasoning` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)